scripts/format.sh

```diff
@@ -16,43 +16,98 @@
 
 set -eu
 
+function log() { printf -- "** %s\n" "$*" >&2; }
```

Comment:
Same here. Can we align the script with the one in `swift-certificates`?

Reply:
This divergence is useful: this version checks out, builds, and caches an appropriate version of `swift-format` making it easier for new developers to run the formatter (they don't need to know what version to get, how to check it out, etc.).

Reply:
I agree that this here does more. We should bring it over to the other repos IMO. I would love to have an aligned approach across all of our repos.

Reply:
Yeah, @dnadoba and I talked about this a little while ago – I believe he was interested in this scripts ability to checkout/build/cache the formatter.